src/components/Carousel/Carousel.tsx

```diff
@@ -71,6 +71,39 @@ const Carousels: FC<ICarouselsProps> = ({
     }
   }, [prevClick, nextClick]);
 
+  useEffect(() => {
```

Comment:
We should avoid using the DOM API when working with React, as it will slow down our app.
Please rewrite it using React components. You can use [refs](https://react.dev/learn/manipulating-the-dom-with-refs#best-practices-for-dom-manipulation-with-refs) or custom components with props

Reply:
i can't access "slick-slide" class using useRef beacause there are only Ant Design's Carousel component

Reply:
> We should avoid using the DOM API when working with React, as it will slow down our app. Please rewrite it using React components. You can use [refs](https://react.dev/learn/manipulating-the-dom-with-refs#best-practices-for-dom-manipulation-with-refs) or custom components with props

i can't access "slick-slide" class using useRef beacause there are only Ant Design's Carousel component